### ISSUE TITLE
Feature request: "Copy URL in clippord" button.

### ISSUE THREAD

**Issue by Saentist** [braice/MuMuDVB#262]
Is it possible to be added option to copy url in web interface 
for example 
https://www.w3schools.com/howto/howto_js_copy_clipboard.asp

**Comment by braice** [comment]
Hello

You should be able to edit the template of the html page pretty easily.
Then please submit a pull request and I will merge

Brice

**Comment by Saentist** [comment]
Any advice's where to apply modifications?

**Comment by braice** [comment]
yes, please have a look to src/unicast_http.c and src/unicast_http.h

**Comment by Saentist** [comment]
`<input type="text" value="" id="resultsURL" readonly="readonly" onclick="this.select();this.focus();this.select();document.execCommand('copy');alert('Link copied')">`

A'm not HTML Guru but if someone can implement this

**Comment by braice** [comment]
Me neither but I guess you can prototype a static page based on the HTML
you get from MuMuDVB, get it working and share the code !
